backend/Squirrel.Core/Squirrel.Core.BLL/Services/ProjectService.cs

```diff
@@ -26,10 +25,20 @@ public sealed class ProjectService : BaseService, IProjectService
 
     public async Task<ProjectResponseDto> AddProjectAsync(NewProjectDto newProjectDto)
     {
+        var currentUserId = _userIdGetter.GetCurrentUserId();
+
         var projectEntity = _mapper.Map<Project>(newProjectDto.Project);
-        projectEntity.CreatedBy = _userIdGetter.GetCurrentUserId();
+        projectEntity.CreatedBy = currentUserId;
         var createdProject = (await _context.Projects.AddAsync(projectEntity)).Entity;
         await _context.SaveChangesAsync();
+        
+        var userEntity = await _context.Users.FirstOrDefaultAsync(u => u.Id == currentUserId);
+        if (userEntity == null)
+        {
+            throw new EntityNotFoundException(nameof(User), currentUserId);
+        }
+        userEntity.Projects.Add(createdProject);
```

Comment:
I guess you don't need to do this at all. This user in an author of this project, not member